integration_tests/pages/apply/risks-and-needs/physicalHealthPage.ts

```diff
@@ -19,4 +19,30 @@ export default class PhysicalHealthPage extends ApplyPage {
       }),
     )
   }
+
+  describePhysicalNeeds = (): void => {
```

Comment:
how much value is there in separating out all of these methods, vs just having e.g. an answerQuestions() method?

Reply:
I _think_ it will make maintenance easier: this page consists of 5 questions, if 2 of the 5 are changing during a review then we update 2 of the 5 step helper functions...